src/Uno.UWP/System/MemoryManager.Android.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Android.OS;
+using Android.App;
+using Uno.UI;
+
+namespace Windows.System
+{
+	partial class MemoryManager
+	{
+		public static ulong AppMemoryUsage
+		{
+			get
+			{
+				Debug.MemoryInfo mi = new Debug.MemoryInfo();
+				Debug.GetMemoryInfo(mi);
+				var totalMemory = mi.NativePrivateDirty + mi.OtherSharedDirty;
```

Comment:
a) Why not add also dalvikPrivateDirty?
b) OtherSharedDirty is, according to doc, in 1 KiB pages, not in bytes. So, multiply it by 1024
c) data from Debug.MemoryInfo are for process. Does Uno Android app has only one process? Or should we iterate processes per app?

Reply:
a+b) I think the correct way to do it is using totalPss:  https://developer.android.com/reference/android/os/Debug.MemoryInfo#getTotalPss() and multiply by 1024. To have an exact memory consumption for a Unix process (taking into consideration shared memory) is something there's no direct answer to it.

c) There's a concept of multi-process applications in Android, but you need specific code to support this and involves multiple & "remote" activities. Maybe @jeromelaban is more proficient on this, but I doubt that Uno has even remote interest on supporting any of this. And if I remember correctly after Android M (?) the OS provides you with a single PID when you ask for your application PIDs. So, IMHO I think this method method is the best bet to provide the information of the memory consumption of an Application.

Reply:
That's correct, multi-process is not of interest at this point, particularly when matching this particular WinRT API. If an application needs fined grained information, it'll need to go through the native api directly.